src/asymmetric_crypto/curve_25519/x25519/key_pair.rs

```diff
@@ -0,0 +1,22 @@
+use rand_core::CryptoRngCore;
```

Comment:
You should use the reexport version:
```suggestion
use crate::reexport::rand_core::CryptoRngCore;
```